src/Navigation/Education/index.js

```diff
@@ -1,9 +1,14 @@
 import React from "react";
+import "./index.css";
+import EducationContent from "./EducationContent";
+import EducationHeader from "./EducationHeader";
+import data from "../../resume.json";
```

Comment:
Pass education data to `Education` component from parent.
If we just import data in App component and pass it to all these section components, then we are not making a hard dependency on these section components.
But if we import data in all section components, we are making hard dependency of data in all these section components.